Sources/BraintreePayPal/Public/BraintreePayPal/BTPayPalClient.h

```diff
@@ -1,11 +1,6 @@
-#if __has_include(<Braintree/BraintreePayPal.h>)
-#import <Braintree/BTPayPalCheckoutRequest.h>
-#import <Braintree/BTPayPalVaultRequest.h>
-#else
-#import <BraintreePayPal/BTPayPalCheckoutRequest.h>
-#import <BraintreePayPal/BTPayPalVaultRequest.h>
-#endif
+#import <Foundation/Foundation.h>
 
+typedef NS_ENUM(NSInteger, BTPayPalPaymentType);
```

Comment:
🧶 : Maybe a note mentioning this will be removed?

Reply:
This enum still exists! It's in `BTPayPalRequest.swift`. This is just us forward declaring it to use in this Obj-C file. Once this is converted to Swift, `BTPayPalPaymentType` will still exist in the same way as it does currently we just won't need to forward declare it here. Really just exists here because of how exposing Swift in Obj-C needs to be done.

Reply:
Should we add a TODO to remove this fwd declare then?

Reply:
It will be removed when we convert `BTPayPalClient` to Swift in a follow up PR. The PayPal conversion is all going into a feature branch so will get cleaned up as we convert the remaining files.